simulator/README.md

```diff
@@ -70,3 +70,20 @@ If a program doesn't require to display a window and only needs to export one or
 `SimulatorDisplay` can also be converted to an `image` crate `ImageBuffer` by using the
 `to_image_buffer` method. The resulting buffer can then be used to save the display content to
 any format supported by `image`.
+
+# Usage without SDL2
+
+The `with-sdl` feature can be disabled by adding `default-features = false` to the dependency. This
+will remove the requirement on SDL2 build files being present and removes the `OutputSettings::output_to_display` and `OutputSettings::pixel_pitch` methods. For example:
```

Comment:
`OutputSettings::output_to_display` and `OutputSettings::pixel_pitch` are internal functions and don't need to be mentioned here IMO.
```suggestion
When the simulator is used in headless/CI environments, that don't require showing a window, the SDL2 support can be disabled. This removes the requirement of SDL2 being installed on the target machine, but still allows the simulator to be used to generate images.

The `with-sdl` feature is enabled by default and can be disabled by adding `default-features = false` to the dependency:
```